specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/quota.json

```diff
@@ -0,0 +1,1296 @@
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The quota request ID. To check the request status, use the **id** value in a [Quota Request Status](https://docs.microsoft.com/en-us/rest/api/reserved-vm-instances/quotarequeststatus/get) GET operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Operation ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Quota request status.",
+          "$ref": "#/definitions/QuotaRequestStatusDetails",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "QuotaRequestStatusDetails": {
+      "description": "Quota request status details.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Quota request status.",
+          "$ref": "#/definitions/QuotaRequestState",
+          "readOnly": true
+        },
+        "message": {
+          "description": "User-friendly message.",
+          "type": "string",
+          "readOnly": true
+        },
+        "limit": {
+          "description": "Quota limit.",
+          "type": "integer",
+          "format": "int32",
+          "readOnly": false
+        },
+        "currentValue": {
+          "description": "Usage information for the current resource.",
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true
+        },
+        "unit": {
+          "description": " The quota limit units, such as Count and Bytes. When requesting quota, use the **unit** value returned in the GET response in the request body of your PUT operation.",
+          "type": "string",
+          "readOnly": false
+        },
+        "name": {
+          "description": "Resource name provided by the resource provider. Use this property name when requesting quota.",
+          "$ref": "#/definitions/ResourceName"
+        },
+        "resourceType": {
+          "description": "Resource type name.",
+          "$ref": "#/definitions/ResourceTypesName"
+        },
+        "quotaPeriod": {
+          "description": "The time period over which the quota usage values are summarized. For example:\r\n*P1D (per one day)\n*PT1M (per one minute)\n*PT1S (per one second).\r\nThis parameter is optional because, for some resources like compute, the period is irrelevant.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Additional properties for the specific resource provider.",
+          "type": "object"
+        }
+      }
+    },
+    "QuotaRequestDetails": {
+      "type": "object",
+      "description": "List of quota requests with details.",
+      "x-ms-azure-resource": true,
```

Comment:
> "x-ms-azure-resource": true, [](http://example.com/codeflow?start=5&length=29)

This doesn't need to be true since it is used to model the response of async operation status api.

Reply:
Will remove it.